Clients/Xamarin.Interactive.Client.Web/ClientApp/components/PackageSearch.tsx

```diff
@@ -6,6 +6,7 @@ import { PrimaryButton } from 'office-ui-fabric-react/lib/Button'
 import { Panel, PanelType } from 'office-ui-fabric-react/lib/Panel'
 import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox'
 import { Icon } from 'office-ui-fabric-react/lib/Icon'
+import 'isomorphic-fetch'
```

Comment:
Where is this coming from? Were we already pulling it in? Does this only kick in for IE11 or does it replace the native implementation in real browsers?

Reply:
We were already pulling it in. Although, I'm not really sure why we are using it instead of Github's `whatwg-fetch` (https://github.com/github/fetch), which `isomorphic-fetch` wraps.

Anyway, as I understand it, polyfills like this don't replace anything if the API is already defined. See https://github.com/github/fetch/blob/master/fetch.js#L511.

Reply:
I would love to know what is pulling it in.

Reply:
In any case, you should specify it explicitly since we explicitly need it.

Reply:
Oh, looks like we already had it. @bojanrajkovic?

Reply:
`isomorphic-fetch` is basically a wrapper that makes sure both the node-side and the client-side get working `fetch` support (hence "isomorphic"). But we don't have a node side. So I think we can just pull in `whatwg-fetch` and have an identical experience with slightly less indirection.

@bojanrajkovic  any comments?

Reply:
It's coming from draftjs, per `npm ll`:

```
├─┬ draft-js@0.10.5
│ │ A React framework for building text editors.
│ │ git+https://github.com/facebook/draft-js.git
│ │ http://draftjs.org/
│ ├─┬ fbjs@0.8.16
│ │ │ A collection of utility libraries used by other Facebook JS projects
│ │ │ git+https://github.com/facebook/fbjs.git
│ │ │ https://github.com/facebook/fbjs#readme
│ │ ├── core-js@1.2.7
│ │ │   Standard library
│ │ │   git+https://github.com/zloirock/core-js.git
│ │ │   https://github.com/zloirock/core-js#readme
│ │ ├── isomorphic-fetch@2.2.1 deduped
│ │ │   Isomorphic WHATWG Fetch API, for Node & Browserify
│ │ │   git+https://github.com/matthew-andrews/isomorphic-fetch.git
│ │ │   https://github.com/matthew-andrews/isomorphic-fetch/issues
```

Reply:
We also have it as a dev-dependency, that was added in 4e38514b.

Reply:
Oh but of course it is coming from `fbjs`. Utility libraries pulling in utilities pulling in utilities pulling in operating systems.  🙄